### PR TITLE
Fix transparent header in issues view.

### PR DESCRIPTION
**Pull request by inuwali**
As noted in the comment thread for the issue, I had to add back in `IssuesContainerViewController.setContentInsets()`; its removal caused a 20-point gap to appear between the header and the table view content.

### CODE REVIEW

**Comment by mccarron** [comment]
This looks good to me and glad it fixes the issues you were seeing. 

Gotta figure out the merging order of this with the iPad changes. @subdigital @bengottlieb what are your thoughts? Merge this in first then refactor iPad?

**Comment by subdigital** [comment]
Yes, this should go in first, as we have some loose ends to work on w/ the iPad app.

File: FiveCalls/FiveCalls/IssuesContainerViewController.swift [hunk 1]
```diff
@@ -38,15 +39,40 @@ class IssuesContainerViewController : UIViewController, EditLocationViewControll
         
         issuesVC.didMove(toParentViewController: self)
         issuesViewController = issuesVC
+        
+        let effectView = UIVisualEffectView(frame: headerContainer.bounds)
+        effectView.translatesAutoresizingMaskIntoConstraints = false
+        effectView.effect = UIBlurEffect(style: .light)
```
Comment (subdigital) [inline FiveCalls/FiveCalls/IssuesContainerViewController.swift]: I like to extract these into lazy properties, like this:

```swift
lazy var effectView: UIVisualEffectView = {
    let effectView = ...
    // more setup
    return effectView
}
```

Then you can just use it in `viewDidLoad` or wherever and they get set up properly.

File: FiveCalls/FiveCalls/IssuesContainerViewController.swift [hunk 2]
```diff
@@ -38,15 +39,40 @@ class IssuesContainerViewController : UIViewController, EditLocationViewControll
...
+            effectView.contentView.topAnchor.constraint(equalTo: headerView.topAnchor),
+            effectView.contentView.bottomAnchor.constraint(equalTo: headerView.bottomAnchor),
+            effectView.contentView.leftAnchor.constraint(equalTo: headerView.leftAnchor),
+            effectView.contentView.rightAnchor.constraint(equalTo: headerView.rightAnchor),
+            
+            headerContainer.topAnchor.constraint(equalTo: effectView.topAnchor),
+            headerContainer.bottomAnchor.constraint(equalTo: effectView.bottomAnchor),
+            headerContainer.leftAnchor.constraint(equalTo: effectView.leftAnchor),
+            headerContainer.rightAnchor.constraint(equalTo: effectView.rightAnchor)
+            ])
```
Comment (subdigital) [inline FiveCalls/FiveCalls/IssuesContainerViewController.swift]: Should probably extract all this header setup into a method... maybe `setupBlurHeader()` ?

File: FiveCalls/FiveCalls/IssuesContainerViewController.swift (hunk 1 above)
Comment (inuwali) [inline FiveCalls/FiveCalls/IssuesContainerViewController.swift]: I'm a Swift newbie so I had to try a few things before it worked... Needed to add () after the closing curly brace.

File: FiveCalls/FiveCalls/IssuesContainerViewController.swift (hunk 2 above)
Comment (inuwali) [inline FiveCalls/FiveCalls/IssuesContainerViewController.swift]: Makes sense.

Comment (inuwali) [inline FiveCalls/FiveCalls/IssuesContainerViewController.swift]: BTW I was wondering why you didn't set up the container/contained relationship directly in the storyboard... Figured you had a reason but didn't investigate.

**Comment by subdigital** [comment]
Thanks, @inuwali!

**Pull request merged**